Modules/Core/Common/test/itkBSplineInterpolationWeightFunctionTest.cxx

```diff
@@ -154,7 +164,7 @@ itkBSplineInterpolationWeightFunctionTest(int, char *[])
     IndexType startIndex1;
     IndexType startIndex2;
 
-    double testFailed = false;
+    bool testFailed = false;
```

Comment:
Please consider moving this adjustment into a commit, as it appears unrelated to the constexpr linkage issue. (Following the "One Commit. One Change" rule.) If you want to fix `double testFailed = false`, please do it on both places (line 167 _and_ line 70) .